docs/tutorial/05-non-fungible-tokens-2.mdx

```diff
@@ -29,22 +29,21 @@ a full implementation for **Non-Fungible Tokens (NFTs)**.
 ---
 
 <Callout type="success">
-
-Open the starter code for this tutorial in the Flow Playground:
-
-<a href="https://play.onflow.org/af553a60-7b73-4e2e-b145-80a7c7e088dc" target="_blank">
-  https://play.onflow.org/af553a60-7b73-4e2e-b145-80a7c7e088dc
-</a> <br/>
-The tutorial will ask you to take various actions to interact with this code.
-
+  Open the starter code for this tutorial in the Flow Playground:
+  <a 
+    href="https://play.onflow.org/af553a60-7b73-4e2e-b145-80a7c7e088dc" 
```

Comment:
It looks like this link is the same as the old one. Did you not have to make any changes to it?

Reply:
No changes. I was just updating the formatting to match the other tutorials